src/EnergyPlus/PollutionModule.hh

```diff
@@ -446,1139 +324,50 @@ namespace PollutionModule {
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0),
-        PollutionModule::CoefficientProps(static_cast<int>(PollutionModule::PollFactor::OtherFuel2),
-                                          false,
-                                          1.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0.0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0,
-                                          0),
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0};
-    PollutionModule::FuelTypeProps FuelType;
```

Comment:
omg...

Reply:
The level of copy-paste in this module was egregious. It's bad in a few other modules, but this is the only one I have seen where it was literally 10X bad.